lib/templates/deploy.bash.ruby.template

```diff
@@ -1,6 +1,12 @@
 # Rails Helpers
 # -----------
 
+setLatestRubyVersion(){
+    # Hardcoded to 2.3.3 for now until we support more versions
+	LATEST_RUBY_VERSION=2.3.3
```

Comment:
I've been looking into how to regex parse rbenv output to grab the latest version but I haven't followed through 100% because it seems like we won't be adding more than 2.3.3 for a while. But it is theoretically possible for sure. It's a placeholder function for later implementation.